app/views/user_mailer/claim_notification.html.erb

```diff
@@ -1,17 +1,17 @@
 <% content_for :message do %>
   <h2 class="heading">
-    Hello from the Archive of Our Own!
+    <%= t(".hello") %>
   </h2>
 
-  <p>You're receiving this e-mail because you had works in a fanworks archive that has been imported by <%= style_link('Open Doors', 'http://opendoors.transformativeworks.org/') %> into the <%= style_link('Archive of Our Own', root_url) %> (AO3). Because this e-mail address is connected to one registered on the imported archive, the associated fanworks (listed below) have been automatically added to your AO3 account.</p>
+  <p><%= t(".html.part1", open_doors: style_link(t(".open_doors"), 'http://opendoors.transformativeworks.org'), ao3: style_link(t(".ao3"), root_url )).html_safe %></p>
 
-  <p>You can read announcements about recent archive moves at <%= style_link('AO3 News', 'http://archiveofourown.org/admin_posts?tag=18') %>, and find additional information on Open Doors' <%= style_link('FAQ page', 'http://opendoors.transformativeworks.org/faq') %> or <%= style_link('tutorials page', 'http://opendoors.transformativeworks.org/tutorials') %>. For any questions not answered in the FAQ, tutorials, or this e-mail, please <%= support_link('contact AO3 Support') %>.</p>
+  <p><%= t(".html.part2", ao3_news: style_link(t(".ao3_news"), 'http://archiveofourown.org/admin_posts?tag=18'), open_doors_faq: style_link(t(".open_doors_faq"), 'http://opendoors.transformativeworks.org/faq'), tutorial_page: style_link(t(".tutorials_page"), 'http://opendoors.transformativeworks.org/tutorials'), support: support_link(t(".contact_support"))).html_safe %></p>
 
-  <p>If this is a mistake and these are not your works, please don't delete them! Please <%= style_link('contact Open Doors', 'http://transformativeworks.org/contact/open%20doors') %> and we will sort it out.</p>
+  <p><%= t(".html.part3", open_doors: style_link(t(".contact_open_doors"), 'http://opendoors.transformativeworks.org/contact-open-doors/')).html_safe %></p>
 
-  <p>Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please <%= support_link('contact Support') %>.</p>
+  <p><%= t(".html.part4", support: support_link(t(".contact_support"))).html_safe %></p>
 
-  <p>These works were written under the e-mail: <%= @external_email %></p>
+  <p><%= t("user_mailer.claim_notification.part5", email: @external_email).html_safe %></p>
```

Comment:
How come we need the user_mailer.claim_notification part here? I see it's included in other places in this file and the the text email as well.